app/views/includes/analytics.nunjucks

```diff
@@ -1,46 +1,79 @@
   <script>
-    // WebTrends SmartSource Data Collector Tag v10.4.23
-    // Copyright (c) 2016 Webtrends Inc.  All rights reserved.
-    // Tag Builder Version: 4.1.3.5
-    // Created: 2016.01.13
-    window.webtrendsAsyncInit=function(){
-      var dcs=new Webtrends.dcs().init({
-        dcsid:'{{ WEBTRENDS_ANALYTICS_TRACKING_ID }}',
-        domain:'statse.webtrendslive.com',
-        timezone:0,
-        i18n:true,
-        adimpressions:true,
-        adsparam:'WT.ac',
-        offsite:true,
-        download:true,
-        downloadtypes:'xls,doc,pdf,txt,csv,zip,docx,xlsx,rar,gzip',
-        onsitedoms:'nhs.uk',
-        fpcdom:'.nhs.uk',
-        plugins:{
-          hm:{src:'//s.webtrends.com/js/webtrends.hm.js'}
-        }
-      }).track();
-    };
-    (function(){
-      var s=document.createElement('script'); s.async=true; s.src="{{ SITE_ROOT }}/js/vendor/tracking/webtrends.min.js";
-      var s2=document.getElementsByTagName('script')[0]; s2.parentNode.insertBefore(s,s2);
-    }());
+    function enableWebtrends() {
+{% if WEBTRENDS_ANALYTICS_TRACKING_ID %}
+      // WebTrends SmartSource Data Collector Tag v10.4.23
+      // Copyright (c) 2016 Webtrends Inc.  All rights reserved.
+      // Tag Builder Version: 4.1.3.5
+      // Created: 2016.01.13
+      window.webtrendsAsyncInit=function(){
+        var dcs=new Webtrends.dcs().init({
+          dcsid:'{{ WEBTRENDS_ANALYTICS_TRACKING_ID }}',
+          domain:'statse.webtrendslive.com',
+          timezone:0,
+          i18n:true,
+          adimpressions:true,
+          adsparam:'WT.ac',
+          offsite:true,
+          download:true,
+          downloadtypes:'xls,doc,pdf,txt,csv,zip,docx,xlsx,rar,gzip',
+          onsitedoms:'nhs.uk',
+          fpcdom:'.nhs.uk',
+          plugins:{
+            hm: {src:'//s.webtrends.com/js/webtrends.hm.js'},
+            id_migrate: { src: '//s.webtrends.com/js/webtrends.idmigrate.js', waitForCallback: true },
+          }
+        }).track();
+      };
+      (function(){
+        var s=document.createElement('script'); s.async=true; s.src="{{ SITE_ROOT }}/js/vendor/tracking/webtrends.min.js";
+        var s2=document.getElementsByTagName('script')[0]; s2.parentNode.insertBefore(s,s2);
+      }());
+{% endif %}
+    }
   </script>
-
+{% if WEBTRENDS_ANALYTICS_TRACKING_ID %}
   <noscript>
-    <img alt="dcsimg" id="dcsimg" width="1" height="1" src="//statse.webtrendslive.com/{{ WEBTRENDS_ANALYTICS_TRACKING_ID }}/njs.gif?dcsuri=/nojavascript&amp;WT.js=No&amp;WT.tv=10.4.23&amp;dcssip=.nhs.uk"/>
+    <img alt="dcsimg" id="dcsimg" width="1" height="1" src="//statse.webtrendslive.com/{{ WEBTRENDS_ANALYTICS_TRACKING_ID }}/njs.gif?dcsuri=/nojavascript&amp;WT.js=No&amp;WT.tv=10.4.23&amp;dcssip=.nhs.uk&amp;dcsipa=1"/>
   </noscript>
 {% endif %}
+
+<script>
+  function CookiebotCallback_OnAccept() {
+    if (Cookiebot.consent.statistics) { 
+      enableGoogleAnalyticsUniversal(); 
+      enableWebtrends();
+      enableHotjar();
```

Comment:
Hotjar is a `preference` consent rather than statistics - I also suggest splitting this in two partials - one that goes in `head` (Hotjar + Cookiebot can also live in that one) and one that lives in `body` (webtrends + GA)

Reply:
Thanks for the info, only a single `CookiebotCallback_OnAccept` can be defined though, so we have to stick with one partial.
I'll move the Hotjar logic to to a `preferences` consent check.

Reply:
I guess my questions is if hotjar will work if not in head.

Reply:
Hotjar is put into head to 'work as soon as possible' https://help.hotjar.com/hc/en-us/articles/115009336727-Manually-Adding-the-Tracking-Code. 
In the new implementation we have to wait for consent before Hotjar is enabled so it needs to be loaded dynamically. The downside is we may miss a bit of tracking while the code fires, but I don't see any way around that.